### ISSUE TITLE
adapt-contrib-media aria-label not attached to center play button

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt_framework#3137]
### Subject of the issue/enhancement/features
The play button over the video is picked up by the reader, but it doesn't announce it, i.e. I navigate to the button using the keyboard but the reader doesn't say anything when I tab into that button. That part of the DOM is controlled by a third party plugin so I'm not sure how simple this would be to add.

### Your environment
* v5.11.0
* All Browsers
* Windows

### Expected behaviour
Both media play buttons (center play & scrub bar play), should read from the same aria-label

**Comment by oliverfoster** [comment]
@joe-replin which screen reader did you use?

**Comment by joe-replin** [comment]
Hello! Apologies, as I missed the notification. This was tested in JAWS 2020.
